src/components/ScrollTo/ScrollTo.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Component from './ScrollTo';
+
+export default {
+    title: 'Components/ScrollTo',
+    component: Component,
+};
+
+function Spam() {
+    const toAdd = document.createElement('p');
+    toAdd.innerHTML = '';
+    toAdd.innerHTML = '<br/>';
+
+    // eslint-disable-next-line
+    toAdd.innerHTML = toAdd.innerHTML + 'spam' + '<br/>';
+    document.body.appendChild(toAdd);
+}
```

Comment:
Does this mean the user has to click on this button X number of times to start the scrolling process?  We could add a large amount with one click via a loop.

Reply:
Yea, let me redo this one to add a for loop

Reply:
I added a loop but the problem is that `ScrollToBottom`  only scrolls to the first `spam` and all the other storybooks now have `spam` at the bottom. It goes away with a refresh, still working on fixing it.

Reply:
That's really strange.  We can go over that in today's meeting too.